### ISSUE TITLE
Question about GraphCL

### ISSUE THREAD

**Issue by ZhaoYuTJPU** [Shen-Lab/GraphCL#54]
Hello, may I ask whether the various hyperparameter Settings in the unsupervised experiment section of your paper have been published? I didn't find it in the unsupervised_TU

**Comment by yyou1996** [comment]
Hi @ZhaoYuTJPU,

Sorry for the late reply. As I recall, HPO for unsupervised_TU should be learning rate in https://github.com/Shen-Lab/GraphCL/blob/master/unsupervised_TU/go.sh from {0.01, 0.001, 0.0001}.
